pkg/resources/metrics.go

```diff
@@ -399,9 +357,59 @@ func ReconcilePostgresFreeStorageAlerts(ctx context.Context, client k8sclient.Cl
 	return nil
 }
 
+func reconcilePostgresCPUUtilizationAlerts(ctx context.Context, client k8sclient.Client, inst *v1alpha1.RHMI, cr *crov1.Postgres) error {
+	// dont create the alert if we are using in cluster storage
+	if strings.ToLower(inst.Spec.UseClusterStorage) == "true" {
+		logrus.Info("skipping postgres free storage alert creation, useClusterStorage is true")
+		return nil
+	}
+
+	alertName := "PostgresCPUHigh"
+	ruleName := "postgres-cpu-high"
+	alertDescription := "the postgres instance {{ $labels.instanceID }} for product {{ $labels.productName }} has been using {{ $value }}% of available CPU for longer than 5 minutes"
+	labels := map[string]string{
+		"severity": "warning",
+	}
+
+	alertExp := intstr.FromString("cro_postgres_cpu_utilization_average > 90")
+
+	_, err := reconcilePrometheusRule(ctx, client, ruleName, cr.Namespace, alertName, alertDescription, sopUrlPostgresCpuUsageHigh, alertFor5Mins, alertExp, labels)
```

Comment:
discussing this on https://github.com/integr8ly/integreatly-operator/pull/1102 maybe push the alert out to 15 min cpu would make the alert more robust against cpu spikes.
```suggestion
	_, err := reconcilePrometheusRule(ctx, client, ruleName, cr.Namespace, alertName, alertDescription, sopUrlPostgresCpuUsageHigh, alertFor15Mins, alertExp, labels)
```